contracthashtool.c

```diff
@@ -170,7 +170,7 @@ int main(int argc, char* argv[]) {
 	if (mode == 0x2 && !priv_key_str)
 		USAGEEXIT("No private key specified\n");
 
-	secp256k1_context_t *secp256k1_ctx = secp256k1_context_create((unsigned int) -1);
+	secp256k1_context_t *secp256k1_ctx = secp256k1_context_create(SECP256K1_CONTEXT_SIGN | SECP256K1_CONTEXT_VERIFY);
```

Comment:
Can we get the secp256k1 header to specify which flags are needed for exactly which functions? The reason I went with -1 is that it is not at all obvious which context flags are needed for things like pubkey_tweak_add without reading the code.